decidim-proposals/app/views/decidim/proposals/versions/index.html.erb

```diff
@@ -1,34 +1,8 @@
 <div class="row">
-  <div class="small-12 columns">
-    <h2 class="heading2">
-      <%= t("changes_at_title", scope: "decidim.proposals.versions", title: versioned_resource.title) %>
-    </h2>
-
-    <h3 class="section-heading">
-      <%= t(".title") %>
-    </h3>
-
-    <div class="row">
-      <div class="mediumlarge-8 columns">
-        <div class="section">
-          <div class="card card--list">
-            <% versioned_resource.versions.each_with_index do |version, index| %>
-              <%= render partial: "version", locals: { version: version, index: index } %>
-            <% end %>
-          </div>
-        </div>
-      </div>
-
-      <div class="columns section mediumlarge-4 large-3">
-        <div class="card extra definition-data">
-          <div class="definition-data__item versions_count">
-            <span class="definition-data__title"><%= t("versions.stats.number_of_versions", scope: "decidim.proposals") %></span>
-            <%= versioned_resource.versions.count %>
-
-            <%= link_to back_to_resource_path_text, back_to_resource_path, class: "button button--sc hollow secondary small expanded" %>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
+  <%= cell(
+    "decidim/versions_list",
+    versioned_resource,
+    version_path: proc { |version_index| url_for(action: :show, id: version_index) },
+    i18n_scope: back_to_resource_path_text_scope
```

Comment:
I'm not very sure about this, but how about replacing this function call with `"decidim.proposals.versions.#{item_name.to_s.pluralize}"`?

Reply:
Done!